src/3d/qgs3dmapscene.h

```diff
@@ -214,6 +214,32 @@ class _3D_EXPORT Qgs3DMapScene : public QObject
      */
     Q_DECL_DEPRECATED static QMap< QString, Qgs3DMapScene * > openScenes() SIP_DEPRECATED;
 
+    /**
+     * Enables OpenGL clipping based on the planes equations defined in \a clipPlaneEquations.
+     * The number of planes is equal to the size of \a clipPlaneEquations.
```

Comment:
An OpenGL implementation ensures that there are at least 8 clipping planes as stated in https://doc.qt.io/qt-5/qt3drender-qclipplane.html#details

I changed it so that it only keeps the first 8 planes.